### PR TITLE
entire site doc edits

### PR DESCRIPTION
**Pull request by rosemarymarano**
Complete documentation review and edit. Huge effort to homogenize all terminology and align the docs with the Oracle Style Guideline standards.

### CODE REVIEW

File: docs-source/content/userguide/cicd/tools.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ interesting.  A good way to get a jumpstart on a model is to use the
 `discoverDomain` tool in WDT which builds a model based on an existing domain.
 
 {{% notice note %}}
-A Model in Image domain takes complete advantage of the WDT tool. It let's
+A Model in Image domain completely takes advantage of the WDT tool. It let's
```
Comment (markxnelson) [inline docs-source/content/userguide/cicd/tools.md:25]: awkward ;p

Comment (rjeberhard) [reply docs-source/content/userguide/cicd/tools.md:25]: lol

Comment (rosemarymarano) [reply docs-source/content/userguide/cicd/tools.md:25]: I agree (Tom's original wording, reworded). I will remove the word "completely", completely. Does that work for you?

Comment (rjeberhard) [reply docs-source/content/userguide/cicd/tools.md:25]: Works for me.

Comment (markxnelson) [reply docs-source/content/userguide/cicd/tools.md:25]: completely

**Pull request merged**